.rubocop.yml

```diff
@@ -0,0 +1,20 @@
+AllCops:
+  NewCops: enable
+  TargetRubyVersion: 2.7
+
+Style/StringLiterals:
+  Enabled: true
+  EnforcedStyle: double_quotes
```

Comment:
@javierg Double quotes It is now the ruby standard. It helps standarize the use between different programming languages.

Reply:
I know there is a switch to use double quotes more on the ruby world, just remember that in ruby single and double quotes behave different. SO there are valid use cases for single quotes, so enforcing them like this can have undesired side effects IMO, is better to remove this check

Reply:
Removing the check can lead to an inconsistent use of strings in the project. The examples you sent to me via DM can be done using double quotes.

Reply:
OK, not strong opinions here,  I think the single/double quote debate is not settle yet, but I dont go strong on one way or another, I find valid uses in the same code base for both options, I prefer to use `''`. over `%{}`, but for interpolation `""` is the best option, so carry on, we can continue this debate on other forums